### PR TITLE
Add LTS v4.2.11 to conda-forge - 2 CVE fixes over 4.2.8, merge to 4.2.x

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Can we create a 4.2.x
Django LTS is currently version 4.2.X , this latest version fixes 2 CVE's
CVE-2024-27351
CVE-2024-24680

https://www.djangoproject.com/download/#supported-versions
https://docs.djangoproject.com/en/5.0/releases/4.2.11/
https://docs.djangoproject.com/en/5.0/releases/4.2.10/
https://docs.djangoproject.com/en/5.0/releases/4.2.9/

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@ocefpaf @astamminger @kwilcox - Could you please review, and consider merging to a new branch. I have included a summary on this ask in the description of this PR. 
Thank you and Regards.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-{% set version = "5.0.3" %}
+{% set version = "4.2.11" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml:1]: This won't work b/c a feedstock already exists. Please send a PR to the feedstock maintainance branch: https://github.com/conda-forge/django-feedstock/tree/4.x

Comment (rxm7706) [reply recipe/meta.yaml:1]: Thank you, I dont know how I missed that branch.
 - https://github.com/conda-forge/django-feedstock/pull/176

Comment (ocefpaf) [reply recipe/meta.yaml:1]: I just created it :-)

But we don't add old versions as feedstock, always as branches in existing ones.

**Pull request closed without merging**